prog/victoria_metrics/victoria_metrics_server_nexus.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+require "forwardable"
+
+require_relative "../../lib/util"
+
+class Prog::VictoriaMetrics::VictoriaMetricsServerNexus < Prog::Base
+  subject_is :victoria_metrics_server
+
+  extend Forwardable
+  def_delegators :victoria_metrics_server, :vm
+
+  def self.assemble(victoria_metrics_resource_id, name)
+    vr = VictoriaMetricsResource[victoria_metrics_resource_id]
+    fail "No existing VictoriaMetricsResource" unless vr
+    Validation.validate_name(name)
+
+    DB.transaction do
+      ubid = VictoriaMetricsServer.generate_ubid
+      vm_st = Prog::Vm::Nexus.assemble_with_sshable(
```

Comment:
Do you want to place all servers of the same VictoriMetricsResource in the same private subnet?

Reply:
I left it as right now this only has single node support. But if we add a functionality to add a new server to existing resource it might be needed. I will add it just in case.

Reply:
I added a private subnet with the resource